go.mod

```diff
@@ -3,8 +3,6 @@ module github.com/maxmind/mm-geofeed-verifier
 go 1.12
 
 require (
+	github.com/TomOnTime/utfutil v0.0.0-20200626160131-0b0178852c8f
```

Comment:
Do we use this package in any of our other software, or, have we done any vetting of this package if it's new for us?

Reply:
In other places, we use https://github.com/dimchansky/utfbom for this same sort of functionality. However, given that this is a standalone repo, I think using the above is fine. I reviewed its code and didn't see anything concerning.